gateway/ln-gateway/tests/integration_tests.rs

```diff
@@ -59,17 +59,65 @@ async fn gatewayd_shows_info_about_all_connected_federations() {
 }
 
 #[tokio::test(flavor = "multi_thread")]
-async fn gatewayd_shows_balance_for_any_connected_federation() -> anyhow::Result<()> {
-    // todo: implement test case
+#[ignore] // balance query needs mint module but that is not configured in fixtures
+async fn gatewayd_shows_balance_for_any_connected_federation() {
+    // TODO: Merge with `gatewayd_shows_info_about_all_connected_federations`
+    // after Issue #2508 is resolved
+    let (_, rpc, fed1, fed2, _) = fixtures::fixtures(None).await;
 
-    Ok(())
+    let id1 = fed1.connection_code().id;
+    let id2 = fed2.connection_code().id;
+
+    connect_federations(&rpc, &[fed1, fed2]).await.unwrap();
+
+    let bal1 = rpc
+        .get_balance(BalancePayload { federation_id: id1 })
```

Comment:
@jkitman, can we support balance queries and assertions with just the dummy module?
this gateway test fails with an error that signals the lack of mint module

Reply:
The gateway needs to be converted to use the NG client.  I suggest focusing on this before we can get these tests to all pass.

Reply:
ack.